--- origsrc/pidgin-encryption-3.0/Makefile.am	2007-05-06 19:38:41.000000000 -0500
+++ src/pidgin-encryption-3.0/Makefile.am	2008-05-19 20:50:17.328125000 -0500
@@ -2,9 +2,9 @@
 
 encryptdir = $(libdir)/pidgin
 
-encrypt_la_LDFLAGS = -module -avoid-version
+encrypt_la_LDFLAGS = -module -avoid-version -no-undefined
 
-encrypt_la_LIBADD = $(NSS_LIBS)
+encrypt_la_LIBADD = $(NSS_LIBS) $(PIDGIN_LIBS)
 
 encrypt_LTLIBRARIES = encrypt.la
 noinst_LIBRARIES =
@@ -97,6 +97,6 @@
 testprog: $(encrypt_la_SOURCES) testmain.c
 	$(CC) -g -Wall -lgdk -lgtk $(AM_CPPFLAGS) $(CPPFLAGS) $(NSS_INCLUDES) $(NSS_CFLAGS) -o testprog testmain.c  $(NSS_LIBS)
 
-SUBDIRS = po intl
+SUBDIRS = po
 
 ACLOCAL_AMFLAGS = -I m4
--- origsrc/pidgin-encryption-3.0/configure.ac	2007-05-06 14:48:15.000000000 -0500
+++ src/pidgin-encryption-3.0/configure.ac	2008-05-19 20:56:58.796875000 -0500
@@ -25,6 +25,8 @@
 
 AC_DEFINE_UNQUOTED(ENC_PACKAGE, "$PACKAGE",
     [Version of pidgin-encryption plugin])
+
+AC_SUBST(GETTEXT_PACKAGE, $PACKAGE)
  
 # before gettexting, in case iconv matters
 case "$host_os" in
@@ -42,8 +44,7 @@
 esac
  
 ALL_LINGUAS="cs da de es fr hu it ja lt nl nn pl pt_BR ro ru sl sv tr uk zh_TW"
-AM_GNU_GETTEXT_VERSION(0.10.40)
-AM_GNU_GETTEXT
+AM_GLIB_GNU_GETTEXT
 
 
 #
